pipeline-runner/internal/jobs/build/common.go

```diff
@@ -78,6 +78,14 @@ func getCommonPodVolumes(componentImages []pipeline.BuildComponentImage) []corev
 				},
 			},
 		},
+		corev1.Volume{
+			Name: git.CloneRepoHomeVolumeName,
+			VolumeSource: corev1.VolumeSource{
+				EmptyDir: &corev1.EmptyDirVolumeSource{
+					SizeLimit: resource.NewScaledQuantity(5, resource.Mega),
+				},
+			},
+		},
```

Comment:
```suggestion
		{
			Name: git.CloneRepoHomeVolumeName,
			VolumeSource: corev1.VolumeSource{
				EmptyDir: &corev1.EmptyDirVolumeSource{
					SizeLimit: resource.NewScaledQuantity(5, resource.Mega),
				},
			},
		},
```
Remove unnecessary type definition

Reply:
Removed